packages/api-clients/open-api/delegationApi.yml

```
@@ -282,6 +282,11 @@ paths:
         - delegation
       operationId: getDelegation
       responses:
+        headers:
+          x-metadata-version:
+            description: The version number of the resource in the read model
+            schema:
+              type: integer
```

suggested
- casing (for current convention)
- definition of the header only once under components
```suggestion
          X-Metadata-Version:
            $ref: '#/components/headers/MetadataVersionHeader'
```